src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int RANDOM_VALUE = 15;
```

Comment:
- random value
- constant
something doesn't add up here :) Call it something like  limit, max-value